### DISCUSSION TITLE
ΠΑΡΑΤΑΣΗ ΓΡΑΠΤΗΣ ΕΡΓΑΣΙΑΣ 2

### DISCUSSION THREAD

**Discussion by Katethoma** [sdy60/syllabus#56] [Show and tell]
Καλησπέρα κε Χωριανόπουλε. Μήπως θα μπορούσατε να αλλάξετε στο study την ημερομηνία κατάθεσης της ΓΕ2 γιατί ακόμη φαίνεται 14 Δεκεμβρίου ?

**Comment by dimidimots** [comment]
Καλησπέρα, ακόμα 14 φαίνεται... @epidrome
